packages/@lwc/signals/tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+    "extends": "../../../tsconfig.json",
+
+    "compilerOptions": {
+        "lib": ["dom", "es2018"]
```

```suggestion
        "lib": ["es2018"]
```

You don't need `dom` here – this package doesn't require it.